lib/glimesh/accounts.ex

```diff
@@ -324,6 +334,12 @@ defmodule Glimesh.Accounts do
     |> Repo.update()
   end
 
+  def update_user(user, attrs) do
```

Comment:
Same as comment above, this is in the account's context and is called update_user but it uses the GCT changeset?